tools/php-cs-fixer/composer.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "project",
+  "require": {
+    "php": "^8.0",
+    "friendsofphp/php-cs-fixer": "^3.17"
+  }
+}
```

Comment:
I am more often now using https://packagist.org/packages/php-cs-fixer/shim in the main `package.json`. It is basically a fake library with zero dependencies that downloads the `.phar` file for you, so that you can run `vendor/bin/php-cs-fixer` without the dependency issue.

Reply:
Was not aware of this shim. Happy to use it.